integration_tests/androidx_test/src/test/java/org/robolectric/integrationtests/axt/EspressoWithPausedLooperTest.java

```diff
@@ -28,7 +28,7 @@ public void setUp() {
   public void launchActivity() {}
 
   @Test
-  public void onIdle_does_not_block() {
+  public void onIdle_doesnt_block() {
```

Comment:
I change that in a recent PR because Android Studio complains that there is a typo in `doesnt`.
Does Google Linter complain too about `does_not`?